pom.xml

```diff
@@ -114,6 +114,18 @@
             <artifactId>aether-spi</artifactId>
             <version>1.0.0.v20140518</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>job-dsl</artifactId>
+            <version>1.35</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>matrix-project</artifactId>
```

Comment:
dsl-plugin require jenkins version >= 1.565
but in that version matrix-project has been cut out from jenkins-core, so we need to depend on matrix-project